community/Makefile

```diff
@@ -10,11 +10,9 @@ reset-docs:
 	git checkout HEAD -- ./sig-list.md ./sig-*/README.md ./wg-*/README.md
 
 generate:
-	go run ./generator/app.go
-	mv OWNERS_ALIASES ..
-
-generate-dockerized:
 	podman run --rm -e WHAT -e GO111MODULE -e GOPROXY -v $(shell pwd):/go/src/app:Z $(GOLANG_IMAGE) make -C /go/src/app generate
```

Comment:
I did a quick test and this does not seem to work:

```
$ podman run --rm -e WHAT -e GO111MODULE -e GOPROXY -v $(pwd):/go/src/app:Z golang:1.12 make -C /go/src/app generate
make: Entering directory '/go/src/app'
podman run --rm -e WHAT -e GO111MODULE -e GOPROXY -v /go/src/app:/go/src/app:Z golang:1.12 make -C /go/src/app generate
make: podman: Command not found
make: *** [Makefile:13: generate] Error 127
make: Leaving directory '/go/src/app'
```

An alternative could be to use the [image from the open services group](https://github.com/open-services-group/community/blob/245511c0abeca780747c903a1368a54091a5dd22/Containerfile):

`podman run --rm --volume $(shell pwd):/workdir:Z quay.io/open-services-group/community-tooling:v0.1.0-dev`